src/Http/Controllers/SwaggerAssetController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+namespace Mtrajano\LaravelSwagger\Http\Controllers;
+
+
+use DateTime;
+use Illuminate\Http\Response;
+use Illuminate\Routing\Controller;
+use RuntimeException;
+
+class SwaggerAssetController extends Controller
+{
+    public function __invoke($asset)
+    {
+        try {
+            $path = laravel_swagger_ui_dist_path($asset);
+
+            $contentType = pathinfo($asset)['extension'] == 'css'
+                ? 'text/css'
+                : 'application/javascript';
+
+            $response = new Response(
+                file_get_contents($path), 200, ['Content-Type' => $contentType]
+            );
+
+            return $response
+                ->setSharedMaxAge(31536000)
+                ->setMaxAge(31536000)
+                ->setExpires(new DateTime('+1 year'));
```

Comment:
we might want to let the user configure these as well, not sure.. 🤔 Although I guess those files shouldn't change that often

Reply:
This files will be got from `swagger-ui` plugin. It'll be changed just if the `swagger-api/swagger-ui` change it.